sw/device/lib/base/log.h

```diff
@@ -87,17 +127,27 @@ void base_log_internal_dv(log_severity_t severity, const char *format, ...);
 //
 // NOTE: the ##__VA_ARGS__ syntax below is a GCC/Clang extension, while
 // "" foo "" is a common C idiom to assert that a macro parameter is a string.
-#define LOG(severity, format, ...)                                       \
-  do {                                                                   \
-    /* The `false` below will eventually be replaced with a device.h     \
-       function `device_is_dv()` or similar, which determines if the     \
-       current device is a DV testbench. */                              \
-    if (false) {                                                         \
-      base_log_internal_dv(severity, "" format "", ##__VA_ARGS__);       \
-    } else {                                                             \
-      base_log_internal_core(severity, __FILE__, __LINE__, "" format "", \
-                             format, ##__VA_ARGS__);                     \
-    }                                                                    \
+#define LOG(_severity, _format, ...)                                          \
+  do {                                                                        \
+    if (kDeviceType == kDeviceSimDV) {                                        \
+      /* Put the file name in the throw away .logs.strings section. */        \
+      __attribute__(                                                          \
+          (section(".logs.strings"))) static const char kFileNameStr[] =      \
+          "" __FILE__ "";                                                     \
```

Comment:
Is this the formatting that clang-format produces? Can you try inserting a newline right before `static`?